composer.json

```diff
@@ -30,38 +30,40 @@
         }
     },
     "require": {
-        "php": "^7.3||~8.0.0",
+        "php": "~7.4.0||~8.0.0||~8.1.0",
         "fig/http-message-util": "^1.1.5",
-        "laminas/laminas-httphandlerrunner": "^1.3.0",
-        "laminas/laminas-stratigility": "^3.3.0",
-        "laminas/laminas-zendframework-bridge": "^1.1.0",
-        "mezzio/mezzio-router": "^3.5",
-        "mezzio/mezzio-template": "^2.1.0",
+        "laminas/laminas-httphandlerrunner": "^1.5",
+        "laminas/laminas-stratigility": "^3.5",
+        "laminas/laminas-zendframework-bridge": "^1.4",
```

Comment:
You can drop the bridge requiremetn entirely, if you make the following additional changes:

- Add a "conflict" rule of `"zendframework/zend-expressive": "*"`
- Remove the "replace" section entirely